org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/JavaSettingsTest.java

```diff
@@ -51,7 +52,8 @@
 @RunWith(MockitoJUnitRunner.class)
 public class JavaSettingsTest extends AbstractCompilationUnitBasedTest {
 
-	private static final String MISSING_SERIAL_VERSION = "org.eclipse.jdt.core.compiler.problem.missingSerialVersion";
+	private static final String MISSING_SERIAL_VERSION = CompilerOptions.OPTION_ReportMissingSerialVersion;
```

Comment:
I'd use JavaCore.COMPILER_PB_MISSING_SERIAL_VERSION and JavaCore.COMPILER_PB_STATIC_ACCESS_RECEIVER

Reply:
Fixed.